bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/actions/BaseNewWizardMenu.java

```diff
@@ -163,10 +163,12 @@ public void dispose() {
 		}
 	}
 
-	/*
+	/**
 	 * Returns the action for the given wizard id, or null if not found.
+	 *
+	 * @since 3.132
```

Comment:
1) Please make the method **package protected**, as we don't expect anyone to overwrite it.
2) Please remove "since" comment after making the method package protected, as it would not be new API anymore.
3) **If** this would be changed to "public" as proposed, this would be wrong version, API tooling shows an error in the IDE. It should be `3.134` and MANIFESt.MF version should be changed from `3.133.100` to `3.134.0`, because new API method would be added.

@laeubi, @HannesWell : looks like API tooling validation on jenkins stopped to work (second time I see it after https://github.com/eclipse-platform/eclipse.platform.ui/issues/2430). 
Could you please check whether is that tycho, PDE or jenkins/maven configuration issue?